.github/labeler.yml

```
@@ -4,61 +4,79 @@
-php:
-  - '**/*.php'
+# Head branch → status hints (works on PRs)
+"status:needs-review":
+  - head-branch: ['^feat/.*', '^fix/.*', '^docs?/.*', '^(chore|build)/.*']
 
-# Add 'tests' label for test file changes.
-tests:
-  - '**/*_test.php'
-  - '**/*Test.php'
+# CI / Ops (glob-based mapping)
+"area:ci":
```

YAMLlint (1.37.1)

[error] duplication of key "area:ci" in mapping

(key-duplicates)